mathesar_ui/src/components/filter-entry/FilterEntry.svelte

```diff
@@ -30,6 +31,9 @@
 
   export let columns: ReadableMapLike<ColumnLikeType['id'], ColumnLikeType>;
   export let getColumnLabel: (column: ColumnLikeType) => string;
+  export let getColumnConstraintType: (
+    column: ColumnLikeType,
+  ) => [ConstraintType] | undefined;
```

Comment:
Within the return type you should use `ConstraintType[]` (array) instead of `[ConstraintType]` (tuple).